tests/flex/test_local_ca_flex.py

```diff
@@ -0,0 +1,101 @@
+import os
+
+import pytest
+import torch
+from torch.nn.attention.flex_attention import create_mask
+
+from hepattn.flex.local_ca import transpose_blockmask
+from hepattn.models.decoder import MaskFormerDecoder
+from hepattn.utils.local_ca import auto_local_ca_mask
+
+
+@pytest.mark.skipif(os.environ.get("TORCH_COMPILE") == "1", reason="Fails with torch.compile/Inductor due to dynamic shapes")
```

Comment:
Might be important to understand which part of the compilation fails here, does it change if you remove the compile decorators as above?

Can we still test a non-compiled version in the CI?